modules/performance-notifications/src/main/java/io/uhndata/cards/performancenotifications/PerformanceUtils.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.uhndata.cards.performancenotifications;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.sling.api.resource.LoginException;
+import org.apache.sling.api.resource.ModifiableValueMap;
+import org.apache.sling.api.resource.PersistenceException;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.resource.ResourceResolverFactory;
+import org.apache.sling.api.resource.ValueMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class PerformanceUtils
+{
+    private static final String LABEL_TODAY = "today";
+    private static final String LABEL_TOTAL = "total";
+    private static final String PERFORMANCE_STATISTICS_PATH = "/PerformanceStatistics/";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PerformanceUtils.class);
+
+    // Hide the utility class constructor
+    private PerformanceUtils()
+    {
+    }
+
+    /**
+     * Gets the 'today' and 'total' values for a performance statistic.
+     * Returns null if the statistic cannot be found.
+     *
+     * @param resolver a ResourceResolver used to query the JCR
+     * @param statName the name of the performance statistic to query
+     * @return the map of 'today' and 'total' values for the performance statistic or null
+     */
+    public static Map<String, Long> getPerformanceStatistic(ResourceResolver resolver, String statName)
```

Comment:
All the methods in here have the same redundant suffix. I would trim it:

```suggestion
    public static Map<String, Long> get(ResourceResolver resolver, String statName)
```